GitHubReactions.ps1

```diff
@@ -247,6 +282,9 @@ filter Set-GitHubReaction
     .PARAMETER PullRequest
         The pull request number.
 
+    .PARAMETER Comment
+        The comment id.
```

Comment:
Can you capitalize `ID` here and elsewhere when referring to it in documentation?

Reply:
```suggestion
        The comment ID.
```

Reply:
fixed here and everywhere